Modules/Libs/QuestieLib.lua

```diff
@@ -143,9 +144,13 @@ function QuestieLib:GetColoredQuestName(questId, showLevel, showState, blizzLike
         local isComplete = QuestieDB.IsComplete(questId)
 
         if isComplete == -1 then
-            name = name .. " (" .. Questie:Colorize(l10n("Failed"), "red") .. ")"
+            name = name .. " " .. Questie:Colorize("(" .. l10n("Failed") .. ")", "red")
         elseif isComplete == 1 then
-            name = name .. " (" .. Questie:Colorize(l10n("Complete"), "green") .. ")"
+            name = name .. " " .. Questie:Colorize("(" .. l10n("Complete") .. ")", "green")
+
+        -- Quests treated as complete - zero objectives or synthetic objectives
+        elseif isComplete == 0 and QuestieDB:GetQuest(questId).isComplete == true then
```

Comment:
So this call of `QuestieDB:GetQuest(questId).isComplete` is required and can have a different value than what `QuestieDB.IsComplete(questId)` return? @Dyaxler

Reply:
Correct @BreakBB